sdk-base/src/main/java/com/mapbox/maps/plugin/MapProjection.kt

```diff
@@ -0,0 +1,69 @@
+package com.mapbox.maps.plugin
+
+import com.mapbox.maps.MapboxExperimental
+import com.mapbox.maps.plugin.delegates.MapProjectionDelegate
+
+/**
+ * Describes projection map is using.
+ *
+ * MapboxMap supports Mercator and Globe projections.
+ */
+@MapboxExperimental
+sealed class MapProjection {
+
+  /**
+   * Mercator projection.
+   *
+   * @see [Mercator projection description](https://en.wikipedia.org/wiki/Mercator_projection)
+   */
+  object Mercator : MapProjection()
+
+  /**
+   * Globe projection is a custom map projection model for rendering the map wrapped around a full 3D globe.
```

Comment:
model => mode?

Reply:
tbh I copied it from our internal FAQ 😄 
but guess `mode` is indeed better